kod/object/passive/itematt/weapatt/waspellt.kod

```diff
@@ -40,12 +40,12 @@ resources:
    
    iaSpelltype_gone = "Your %s suddenly seems a little more... ordinary."
 
-   iaSpelltype_holy = "The weapon glows with a pure, white light."
-   iaSpelltype_unholy = "The weapon seems to reek of malevolence."
-   iaSpelltype_fire = "The weapon is hot to the touch."
-   iaSpelltype_cold = "It is icy cold to the touch."
-   iaSpelltype_shock = "Nearly invisible currents run the length of the blade."
-   iaSpelltype_acid = "A thin coating of acid covers the length of the blade."
+   iaSpelltype_holy = "  The weapon glows with a pure, white light."
+   iaSpelltype_unholy = "  The weapon seems to reek of malevolence."
+   iaSpelltype_fire = "  The weapon is hot to the touch."
+   iaSpelltype_cold = "  It is icy cold to the touch."
+   iaSpelltype_shock = "  Nearly invisible currents run the length of the blade."
+   iaSpelltype_acid = "  A thin coating of acid covers the length of the blade."
```

Comment:
What's the deal with these spaces?

Reply:
Dumb move on my part. This is to create space between the base description and these modifier strings, but I need to move this down to where they're appended.

Reply:
This has been streamlined in the latest commit. There's a bit of logic that follows to add the maker's string to the description that DOES NOT need this, as the spaces for that string are in `spell.kod:134`.